components/QuestionHeader/QuestionHeader.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import styles from "./QuestionHeader.module.scss";
+
+type PropTypes = {
+    text: string;
+    required?: boolean;
+};
+
+const QuestionHeader = ({ text, required }: PropTypes): JSX.Element => {
+    return (
+        <h3>
+            {text}
+            {required && <span className={styles.required}>*</span>}
```

Comment:
As @BeckyBlake pointed out, the asterisks are the same color as the question text in the Figma which was an oversight on my part. For now we can just go back to using <h3>